### DISCUSSION TITLE
[Config Support]: Motion Sensor timeout

### DISCUSSION THREAD

**Discussion by marsp88** [blakeblackshear/frigate#13970] [Ask A Question] [answered]
### Describe the problem you are having

Hello,
is there a way to configure the timeout of the motion sensor binary sensor in home assistant?
Right now, when motion is detected, it takes at least 30 seconds for the sensor to report clear again once the motion stopped.
Thank you! 

### Version

0.14.1

### Frigate config file

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

CPU (no coral)

**Comment by NickM-27** [comment, marked as answer]
yes, `mqtt_off_delay` attribute of `motion`
